react/src/demo/App.tsx

```diff
@@ -63,8 +64,18 @@ const App: React.FC = () => {
         };
     }, []);
 
+    const commonChildren = (
+        <>
+            <div style={{ height: 200, width: 2000 }} />
+            <div style={{ height: 200, overflow: "auto" }}>
+                <div style={{ height: 300 }} />
+            </div>
+        </>
+    );
+
     return (
         <div>
+            <ScrollArea>{commonChildren}</ScrollArea>
```

Comment:
Remove temporary code, or add `ScrollArea` correctly to demo-app. I assume code should be removed as the `ScrollArea` storybook should be good enough.